src/Microsoft.NetCore.Analyzers/CSharp/Performance/CSharpDoNotUseCountAsyncWhenAnyAsyncCanBeUsed.Fixer.cs

```diff
@@ -160,14 +174,34 @@ private static void GetFixerForGreaterThanOrEqualExpression(BinaryExpressionSynt
             }
         }
 
-        private static void GetFixerForBinaryExpression(InvocationExpressionSyntax invocationExpression, LiteralExpressionSyntax literalExpression, int value, out SyntaxNode expression, out IEnumerable<SyntaxNode> arguments, out bool negate)
+        private static void GetFixerForBinaryExpression(ExpressionSyntax sourceExpression, bool isAsync, LiteralExpressionSyntax literalExpression, int value, out SyntaxNode expression, out IEnumerable<SyntaxNode> arguments, out bool negate)
         {
-            GetExpressionAndInvocationArguments(invocationExpression, out expression, out arguments);
+            GetExpressionAndInvocationArguments(sourceExpression, isAsync, out expression, out arguments);
             negate = (int)literalExpression.Token.Value == value;
         }
 
-        private static void GetExpressionAndInvocationArguments(InvocationExpressionSyntax invocationExpression, out SyntaxNode expression, out IEnumerable<SyntaxNode> arguments)
+        private static void GetExpressionAndInvocationArguments(ExpressionSyntax sourceExpression, bool isAsync, out SyntaxNode expression, out IEnumerable<SyntaxNode> arguments)
         {
+            while (sourceExpression is ParenthesizedExpressionSyntax parenthesizedExpression)
```

Comment:
We can move this loop into an extension method, say `WalkUpParentheses`, if one does not already exist.

Reply:
Where would that extensions go?

Reply:
Possibly on SyntaxNodeExtensions, but you don't need to do it as part of this PR..